include/ljmd.h

```diff
@@ -0,0 +1,90 @@
+/* 
+ * simple lennard-jones potential MD code with velocity verlet.
+ * units: Length=Angstrom, Mass=amu; Energy=kcal
+ *
+ * baseline c version.
+ *
+ * author: Stefano Campanella
+ */
+
+#ifndef __LJMD__
+#define __LJMD__
+
+#include <stdio.h>
+
+/**
+ * @brief generic file- or pathname buffer length 
+ */
+#define BLEN 200
+
+/**
+ * @brief structure to hold the complete information
+ * about the MD system
+ */
+struct _mdsys {
+  int natoms, nfi, nsteps;
+  double dt, mass, epsilon, sigma, box, rcut;
+  double ekin, epot, temp;
+  double *rx, *ry, *rz;
+  double *vx, *vy, *vz;
+  double *fx, *fy, *fz;
+};
+typedef struct _mdsys mdsys_t;
+
+/**
+ * @brief boltzman constant in kcal/mol/K
+ */
+const double kboltz = 0.0019872067;
```

Comment:
This constant will lead to conflicts when linking. Please, do:

1. Use: 

```c
extern const double kboltz;
extern const double mvsq2e;
```

2. Create a file named: "constants.c" with the following contents:

```c
#include "../include/ljmd.h"

const double kboltz = 0.0019872067;
const double mvsq2e = 2390.05736153349;
```